packages/block-editor/src/components/url-input/index.js

```diff
@@ -256,7 +256,8 @@ class URLInput extends Component {
 			id,
 			isFullWidth,
 			hasBorder,
-			renderSuggestions,
+			__experimentalRenderSuggestions,
+			__experimentalOnKeyPress,
```

Comment:
I _think_ @retrofox is going to remove this right Damián?

Reply:
gone